app/controllers/schools/participants_controller.rb

```diff
@@ -65,6 +65,7 @@ def update_email
   def email_used; end
 
   def edit_start_term
+    @cohort = @profile.induction_records.first.school_cohort.cohort
```

Comment:
Should it be `@profile.current_induction_record` ?

Reply:
probably... i think I had in mind about people who maybe changed cohorts but that'd be rare and should still be the current induction record.